getting-started/pull-request-lifecycle.rst

```diff
@@ -1,12 +1,15 @@
+.. _pull-request-lifecycle:
 .. _patch:
+.. _pullrequest:
 
+=============================
 Lifecycle of a Pull Request
```

Comment:
This is off-center according to this PR's header style, but if we revert back to the previous style it doen't matter.